docs/UserGuide.adoc

```diff
@@ -98,20 +102,30 @@ Examples:
 The best part about this is you do not even need to key in the time that you created this loan! LoanBook does the timing for you! Simple isn't it?
 ====
 
+[big red]#List of Parameters#:
+
+`n/`: `PERSON'S NAME` +
+`ic/`: `PERSON'S IC` +
+`p/`: `PERSON'S PHONE NUMBER` +
+`e/`: `PERSON'S EMAIL` +
+`b/`: `BIKE NAME` to be rented +
+`lr/`: `RATE` of the loan, in dollars per hour +
+[`t/`: `TAGS`] (Optional) +
```

Comment:
I think this is OK, because it tells you the value of what each prefix is expecting though.
The addbike one had:
`n/`: `BIKE NAME`, which is consistent to this one.

Reply:
I think we should not format things like `BIKE NAME`, since it's not a code term and especially if it's inconsistent with the name used in the Format specification (like `e/EMAIL` vs `e/`:`PERSON'S EMAIL`). The square brackets are also not formatted, when they have a specific meaning in the command line. Hence my suggestion, to make things clearer by using similar conventions.

Reply:
OK, you raise a good point. I have fixed this issue.